front/lib/models/assistant/conversation.ts

```diff
@@ -247,10 +248,10 @@ export class AgentMessage extends Model<
   declare agentDustAppRunActionId: ForeignKey<
     AgentDustAppRunAction["id"]
   > | null;
-
   declare agentTablesQueryActionId: ForeignKey<
     AgentTablesQueryAction["id"]
   > | null;
+  declare agentProcessActionId: ForeignKey<AgentTablesQueryAction["id"]> | null;
```

Comment:
Seen IRL, this is temporary we will change the relationship in a follow up PR to have the link action > agent message.